src/GraphQL.Tests/StarWars/StarWarsIntrospectionTests.cs

```diff
@@ -369,7 +369,7 @@ public void allows_querying_field_args()
                             'kind': 'SCALAR'
                           }
                         },
-                        'defaultValue': null
+                        'defaultValue': 'null'
```

Comment:
This change is making GraphIQL not able to parse the introspection. Is anyone else having this problem?

Reply:
Note after reverting this change set, graph iql works as expected.

Reply:
@joemcbride should this be sent as a new issue?

Reply:
@holm0563 may need to bump the GraphiQL version to one that supports the null keyword perhaps?

Reply:
@joemcbride i didn't look at the spec but I'd be shocked if its truly supposed to change to 'null' as a string?

Reply:
AH, I see what you're saying.

https://github.com/graphql/graphql-js/blob/4f5e351c2afb28e4ae3a44eae76d45d895ff3aa2/src/utilities/astFromValue.js#L70-L78

The JS project treats `null` different than `undefined` and `NaN`.  We don't have the "luxury" of 3 different "nulls" in C#.  Will have to decide on the best course of action here.